gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -412,4 +285,196 @@ public long countBytes(PublishRequest request) {
           return request.getSerializedSize();
         }
       };
+
+  public static class Builder extends ServiceApiSettings.Builder {
+    private final ImmutableList<ApiCallSettings.Builder> methodSettingsBuilders;
+
+    private SimpleCallSettings.Builder<Topic, Topic> createTopicSettings;
+    private BundlingCallSettings.Builder<PublishRequest, PublishResponse> publishSettings;
+    private SimpleCallSettings.Builder<GetTopicRequest, Topic> getTopicSettings;
+    private PageStreamingCallSettings.Builder<ListTopicsRequest, ListTopicsResponse, Topic>
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'listTopicsSettings' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2381705100/issues/source?bid=3123109&fileBranchId=3225976#l296)